src/stores/views.ts

```diff
@@ -19,6 +19,7 @@ export const useTravelTimesViewStore = defineStore('traveltimes-views', () => {
 })
 
 export const useLineViewsStore = defineStore('line-views', () => {
+  const selectedTravelTime: Ref<TravelTimeModel | null> = ref(null)
```

Comment:
Maybe add a common store containing the selectedTravelTime variable? Because each time we want to add a double arrow on a new view we will have to add the variable selectedTravelTime to the store of this view